bundles/org.openhab.binding.ecotouch/src/main/resources/OH-INF/i18n/ecotouch_de.properties

```diff
@@ -0,0 +1,249 @@
+channel-type.ecotouch.temperature_source_in.label = Quelleneintrittstemperatur
+channel-type.ecotouch.temperature_source_in.description = Quelleneintrittstemperatur
+channel-type.ecotouch.temperature_source_out.label = Quellenaustrittstemperatur
+channel-type.ecotouch.temperature_source_out.description = Quellenaustrittstemperatur
+channel-type.ecotouch.temperature_evaporation.label = Verdampfungstemperatur
+channel-type.ecotouch.temperature_evaporation.description = Verdampfungstemperatur
+channel-type.ecotouch.temperature_suction.label = Sauggastemperatur
+channel-type.ecotouch.temperature_suction.description = Sauggastemperatur
+channel-type.ecotouch.pressure_evaporation.label = Verdampfungsdruck
+channel-type.ecotouch.pressure_evaporation.description = Verdampfungsdruck
+channel-type.ecotouch.temperature_return_set.label = Temperatur R�cklauf Soll
+channel-type.ecotouch.temperature_return_set.description = Temperatur R�cklauf Soll
+channel-type.ecotouch.temperature_return.label = R�cklauftemperatur
+channel-type.ecotouch.temperature_return.description = R�cklauftemperatur
+channel-type.ecotouch.temperature_flow.label = Vorlauftemperatur
+channel-type.ecotouch.temperature_flow.description = Vorlauftemperatur
+channel-type.ecotouch.temperature_condensation.label = Kondensationstemperatur
+channel-type.ecotouch.temperature_condensation.description = Kondensationstemperatur
+channel-type.ecotouch.pressure_condensation.label = Kondensationsdruck
+channel-type.ecotouch.pressure_condensation.description = Kondensationsdruck
+channel-type.ecotouch.temperature_storage.label = Speichertemperatur
+channel-type.ecotouch.temperature_storage.description = Speichertemperatur
+channel-type.ecotouch.temperature_room.label = Raumtemperatur
+channel-type.ecotouch.temperature_room.description = Raumtemperatur
+channel-type.ecotouch.temperature_room_1h.label = Raumtemperatur gemittelt 1h
+channel-type.ecotouch.temperature_room_1h.description = Raumtemperatur gemittelt �ber 1 Stunde
+channel-type.ecotouch.temperature_water.label = Warmwassertemperatur
+channel-type.ecotouch.temperature_water.description = aktuelle Temperatur im Warmwasserspeicher
+channel-type.ecotouch.temperature_pool.label = Pool-Temperatur
+channel-type.ecotouch.temperature_pool.description = Pool-Temperatur
+channel-type.ecotouch.temperature_solarlabel = Solarkollektortemperatur
+channel-type.ecotouch.temperature_solar.description = Solarkollektortemperatur
+channel-type.ecotouch.temperature_solar_flow.label = Solarkreis Vorlauftemperatur
+channel-type.ecotouch.temperature_solar_flow.description = Solarkreis Vorlauftemperatur
+channel-type.ecotouch.position_expansion_valve.label = Ventil�ffnung el. Expansionsventil
+channel-type.ecotouch.position_expansion_valve.description = Ventil�ffnung elektrisches Expansionsventil
+channel-type.ecotouch.power_compressor.label = elektrische Antriebsleitung Verdichter
```

Comment:
Words in labels should be capitalized (except prepositions and so on). See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Please check all.

Reply:
Can you comment on that?